src/reducers/tasksReducer.js

```diff
@@ -163,6 +164,32 @@ export default function(state = initialState, action) {
       });
     }
 
+    case 'MOVE_TASK':
+      const { oldParentGid, newParentGid, taskGid } = action;
+      const newMap = Object.assign({}, state.taskMap);
+      const newParentTask = newMap[newParentGid];
+      const oldParentTask = newMap[oldParentGid];
+      const oldParentSubtaskGids = oldParentTask.subtask_gids.filter(
+        gid => gid !== taskGid
+      );
+      const newSubtaskGids = [...newParentTask.subtask_gids, taskGid];
```

Comment:
So that what redux shows right after the drag-and-drop occurs is the same as what the user will see after a refresh, technically the `subtask_gids` should be ordered by `created` and then by `id`. Think it'd be too slow to do that sorting here?

Reply:
Sure :-)